.cicd/scripts/srw_metric_example.sh

```diff
@@ -50,15 +50,15 @@ conda activate regional_workflow
 set -e -u
```

Comment:
Might want to include `#!/bin/bash` on the first line just like most other shell scripts in the app.

Reply:
@gsketefian , thanks for the comment! I just wonder if this is the [line](https://github.com/clouden90/ufs-srweather-app/blob/ee7f9150a4558f6636fbe674447764688846f204/.cicd/scripts/srw_metric_example.sh#LL1C1-L1C1) you are looking for.

Reply:
@gsketefian As @clouden90 noted above, the Jenkins-based shell scripts all use `#!/usr/bin/env bash`, rather than `#!/bin/bash`.  Using this a safer approach to ensure that the script will run on any of the Tier-1 platforms.  Moving forward, it might be a good idea to replace the current `#!/bin/bash` with `#!/usr/bin/env bash`.